akka-remote/src/main/resources/reference.conf

```diff
@@ -659,13 +659,15 @@ akka {
         # Protocol to use for SSL encryption, choose from:
         # TLS 1.2 is available since JDK7, and default since JDK8:
         # https://blogs.oracle.com/java-platform-group/entry/java_8_will_use_tls
+        # TLS 1.3 is available since JDK 11:
+        # https://www.oracle.com/technetwork/java/javase/11-relnote-issues-5012449.html
         protocol = "TLSv1.2"
 
         # Example: ["TLS_RSA_WITH_AES_128_CBC_SHA", "TLS_RSA_WITH_AES_256_CBC_SHA"]
-        # You need to install the JCE Unlimited Strength Jurisdiction Policy
-        # Files to use AES 256.
+        # If you use a JDK 7 prior to 7u171 or a JDK 8 prior to 8u161 you need to install
```

Comment:
We don't support pre JDK 8 (because Scala) so that part is probably not worth mentioning?

Reply:
I wasn't entirely sure so err'd to the safe side. Thanks for alerting!